service.go

```diff
@@ -302,7 +304,7 @@ func (svc *Service) hasPermission(app *App, event *nostr.Event, requestMethod st
 		if maxAmount != 0 {
 			budgetUsage := svc.GetBudgetUsage(&appPermission)
 
-			if budgetUsage+paymentRequest.MSatoshi/1000 > int64(maxAmount) {
+			if budgetUsage+amount/1000 > int64(maxAmount) {
```

Comment:
The check above needs to be changed

```
if requestMethod == NIP_47_PAY_INVOICE_METHOD
```

Can you add a test for this first and make sure it fails?

I think the check should be changed to `if amount > 0`

Reply:
no longer needed, we decided to use one permission for all payment methods